charts/project-origin-wallet/templates/service-wallet.yaml

```diff
@@ -9,10 +9,14 @@ spec:
   selector:
     app: po-wallet
   ports:
+    - name: rest
+      protocol: TCP
+      port: 5000
+      targetPort: 5000
     - name: grpc
       protocol: TCP
       port: 80
-      targetPort: 80
+      targetPort: 5001
```

Comment:
I just want to be sure that this is not a breaking change. This mean that we can still route REST calls to the "grpc" port and it will work as long as it is HTTP/2 . Correct?

Reply:
Hi, as long as the ingress calls are routed to the port 80 on the service:po-wallet then it will go to port 5001 on the container, which was also changed, look in the appsettings.json.  
I specifically did not change the service-port as not to make a breaking change.